src/http.rs

```diff
@@ -137,22 +183,32 @@ pub async fn run_server(
 	store: Arc<DB>,
 	cfg: RuntimeConfig,
 	_cell_query_tx: SyncSender<CellContentQueryPayload>,
+	counter: Arc<Mutex<u64>>,
 ) {
 	let host = cfg.http_server_host.clone();
 	let port = cfg.http_server_port;
 
 	let get_mode =
 		warp::path!("v1" / "mode").map(move || warp::reply::json(&Mode::from(cfg.app_id)));
+	let counter_clone = counter.clone();
+	let counter_clone1 = counter.clone();
```

Comment:
Do we need both of these clones, or can we just clone in place the `counter` var when passing it to the func?

Reply:
as the values are moving in functions, I don't think without this would work! Still might be wrong